README.md

```diff
@@ -1 +1,143 @@
-# HTML-CSS-JavaScript-capstone-project-web-development-course.
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+
+- [🔭 Future Features](#future-features)
+
+- [🤝 Contributing](#contributing)
+
+- [⭐️ Show your support](#support)
+ 
+- [🙏 Acknowledgements](#acknowledgements)
+
+- [📝 License](#license)
+
+
+# 📖  Capstone Project: HTML/CSS & JavaScript project <a name="about-project"></a>
+
+**First Capstone: HTML/CSS & JavaScript** is website for Web Development curse event project build with HTML, CSS, Javascript , using design guidelines , a mobile first approach, with best practices, Github flow and a descriptive and easy to understand README file.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+- Javascript
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+    <summary>Client</summary>
+        <ul>
+            <li><a  href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+        </ul>
+        <ul>
+            <li><a  href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+        </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+-  Desktop Version
+-  Mobile Version
+-  Home Page
+-  About Page
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://deniodev.github.io/HTML-CSS-JavaScript-Capstone-Project/)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Web browser.
+- Code editor.
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this command: 
+
+```sh
+
+ git clone git@github.com:deniodev/HTML-CSS-JavaScript-Capstone-Project.git
+```
+### Install
+
+Install this project with:
+
+Run this command:
+
+```sh
+ cd my-folder
+ npm install
+```
+### Usage
+
+To run the project, execute the following command:
+
+Open index.html using live server extension.
+
+### Run Tests
+
+-N/A
+
+### Deployment
+
+- [Live Demo Link](https://deniodev.github.io/HTML-CSS-JavaScript-Capstone-Project/)
+
+## 👥 Authors <a name="getting-started"></a>
+
+- 👤 Denio Nhanale
+
+    - GitHub: [@githubhandle](https://github.com/deniodev)
+    - LinkedIn: [LinkedIn](https://www.linkedin.com/in/denionhanale/)
+
+- 👤 [Cindy Shin](https://www.behance.net/adagio07)
+
+## 🔭 Future Features <a name="future-features"></a>
+
```

Comment:
- [ ] [OPTIONAL] Good job implementing the changes asked by the previous reviewer, noticed that `Cindy Shin` is not an author of the project. Kindly make sure you update that 👍